enterprise/payara.tooling/src/org/netbeans/modules/payara/tooling/server/config/PayaraV6.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+-->
+<server>
+    <tools lib="lib">
+        <asadmin jar="client/appserver-cli.jar"/>
+    </tools>
+    <java version="17">
+        <platform version="11"/>
+        <platform version="12"/>
+        <platform version="13"/>
+        <platform version="14"/>
+        <platform version="15"/>
+        <platform version="16"/>
+        <platform version="17"/>
+    </java>
+    <javaee version="10.0.0">
+        <profile version="10.0.0" type="web"/>
+        <profile version="10.0.0" type="full" check="full"/>
```

Comment:
This looks strange - previous version of payara declare support for multiple profiles - was that support never there or was it removed? (I compared with PayaraV5.xml).

I doubt this can be parsed, as the constants (version 10.0.0) are not defined in JavaEEProfile.java and it looks as if it should be. I assume this ties back into general EE support - is the support there yet?

Reply:
Thanks @matthiasblaesing for reviewing the PR, I have set the 9.0.0 and 9.1.0 to Payara 6 profile and add the 10.0.0 version to Jakarta EE 10 support PR (WIP).